packages/react-native-aztec/android/build.gradle

```diff
@@ -30,7 +30,7 @@ plugins {
     id "com.android.library"
     id "org.jetbrains.kotlin.android"
     id "maven-publish"
-    id "com.automattic.android.publish-to-s3"
+    id "com.automatic.android.publish-to-s3"
```

Comment:
I'm not an expert on native apps, but it seems to me that we shouldn't change "automattic" to "automatic" in all of these PRs.

cc @fluiddot @geriux